var/spack/repos/builtin/packages/py-pyedr/package.py

```diff
@@ -15,6 +15,7 @@ class PyPyedr(PythonPackage):
 
     maintainers("RMeli")
 
+    version("0.7.2", sha256="8a02b7d94f97f99083c489568f7816ee8ed37e2efca0c1ba3a2e4b83e932d5b9")
     version("0.7.1", sha256="ad7ccdeb739399acd11a25f2d2413ebb46a54223059a2b902ac604d29fabd767")
 
     depends_on("python@3.6:", type=("build", "run"))
```

Comment:
Optional: Spack only supports `python@3.7:` anyway, so you can remove this line.